### PR TITLE
Enable description of latest announcements, evets and news on homepage

### LINKED ISSUE #53: Description not shown for news, event and announcement at index

### PR DESCRIPTION
**Pull request by mohitkyadav**
Fixes #53

### CODE REVIEW

**Comment by mohitkyadav** [comment]
screenshot 71

**Comment by mohitkyadav** [comment]
screenshot 71

**Comment by singhpratyush** [comment]
@mohitkyadav: Can you fix the link colours too? They look ugly.

**Comment by mohitkyadav** [comment]
@singhpratyush sir, I'm on it.

**Comment by mohitkyadav** [comment]
@singhpratyush  Sir, I've made some changes, take a look.
https://drive.google.com/open?id=0B-v6cSjx7ocYZy1oLTFURnpFLXc

**Comment by mohitkyadav** [comment]
Please take a look https://drive.google.com/open?id=0B-v6cSjx7ocYb3Q3dmZnaTZ6LW8

File: wsgi/odyssy/basic/templates/basic/index.html [hunk 1]
```diff
@@ -71,7 +74,12 @@
                                     </span>
                                 </div>
                                 <!-- Event-Description -->
-                                <div class="event-text"><a href="{% url 'event-view-single' event.id %}">{{ event.name }}</a></div>
+                                <div class="widgets-item-text">
+                                    <a class="widgets-item-title" href="{% url 'event-view-single' event.id %}">{{ event.name }}</a>
+                                    <div class="widgets-item-description">
+                                        {{ event.description|safe|truncatewords_html:"10"|linebreaks }}
```
Comment (singhpratyush) [inline wsgi/odyssy/basic/templates/basic/index.html]: Spaces were supposed to be added, not removed.

Comment (mohitkyadav) [inline wsgi/odyssy/basic/templates/basic/index.html]: In Django documentation there's no space arround `|`. Take a look, search for 
`{% firstof var1 var2|safe var3 "<strong>fallback value</strong>"|safe %}`
https://docs.djangoproject.com/en/1.11/ref/templates/builtins/

**Pull request merged**